dist/pathfora.js

```diff
@@ -2910,7 +2910,19 @@
 
       this.getData(recommendUrl, function (json) {
         var resp = JSON.parse(json);
+
         if (resp.data && resp.data.length > 0) {
+          // append a protocol for urls that are absolute
+          for (var i = 0; i < resp.data.length; i++) {
+            var url = resp.data[i].url;
+            if (url) {
+              var split = url.split('/')[0].split('.');
+              if (split.length > 1) {
+                resp.data[i].url = 'http://' + url;
```

Comment:
I don't remember any resolution from any specific conversations about it, but adding it as a property in the model is a good idea: https://github.com/lytics/lio/issues/9516